.github/workflows/database-migrations.yml

```
@@ -0,0 +1,159 @@
+env:
+  AZURE_RESOURCE_GROUP: web-wizards-rg
+  CONTAINER_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  MYSQL_SERVER_NAME: web-wizards-mysql
+  MYSQL_DATABASE_NAME: webwizardsdb
+  ENVIRONMENT: production
+  AZURE_REGION: southeastasia  # Updated for Bangladesh users
+
+jobs:
+  setup-database:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)